.github/workflows/ci.yml

```diff
@@ -95,6 +95,17 @@ jobs:
       - name: Run tox
         if: matrix.python != '3.8'
         run: tox -e py${{ steps.tox.outputs.replaced }}-aiida_vasp
+
+      - name: Archive error mock calculations
```

Comment:
What is this for?

Reply:
This is trying to solve a issue @espenfl raised regarding the mock-code. It can be difficult to debug the mock calculation because it is run by AiiDA as a separate process. I extended the pytest fixture for setting up AiiDA to print extra information and export calculations as an archive if any failed mock-code calculation is detected. Then this step of the CI uploads the archive as an artefact, which can be imported to analyse the cause later.

Reply:
I see. Thanks @zhubonan for your answer. Many technologies!

Reply:
Yes, I support this addition.